ui/apos/apps/index.js

```diff
@@ -1,10 +1,13 @@
 export default () => {
   let ready = false;
 
-  window.apos.util.onReady(() => {
+  apos.util.onReady(() => {
     if (!ready) {
       ready = true;
-      window.apos.bus.$on('export-download', openUrl);
+      apos.bus.$on('export-download', openUrl);
+      apos.bus.$on('import-started', addBeforeUnloadListener);
```

Comment:
Is it possible to have stacked imports?

Reply:
We'll handle that when we'll have a state manager, it's fucked up rn.